### PR TITLE
Tasks.csv Updated again

### PR DESCRIPTION
**Pull request by Sulaiman001**

### CODE REVIEW

**Comment by RajputUsman** [comment]
@Sulaiman001 Now try to use the **Reviewers** feature here

**Comment by RajputUsman** [comment]
@Sulaiman001 Don't use the title of the PR use the title of the **issue.** Also, see the date standardize formation. i.e - 02/12/2020 use as this **2/12/20**.

File: csvs/tasks.csv [hunk 1]
```diff
@@ -456,6 +456,13 @@ Website,Update Docs for Bank - Confirmation Blocks,956,980,VincentHch,12/05/20,1
...
-thenewboston-js,Implemented post request for bank blocks under primary-validator 😁😁,47,54,nishp77,12/23/2020,2500,FALSE
-Validator,Write test_confirmation_block_history_post,108,130,DucPhamTV,12/18/2020,1000,FALSE
+thenewboston-js,Implement Payment Handler,51,50,sno2,12/23/20,3000,FALSE
+thenewboston-js,Implemented post request for bank blocks under primary-validator 😁😁,47,54,nishp77,12/23/20,2500,FALSE
+Validator,Write test for class ConfirmationBlockHistory to cover creation of block history,108,130,DucPhamTV,12/18/20,1000,FALSE
+Validator,Write a test for function add_block_to_queue to add block to the queue,114,127,DucPhamTV,12/13/20,1000,FALSE
+Validator,Write a test for function head_block_hash_view(), queued_confirmation_blocks_view() and valid_confirmation_blocks_view(),105,126,DucPhamTV,12/13/20,1000,FALSE
+Validator,Write a test for function block_chain_view() and block_queue_view(),104,126,DucPhamTV,12/13/20,1000,FALSE
+Validator,Create a test to create a bank using post method,102,126,DucPhamTV,12/13/20,1000,FALSE
+Validator,Write test for class ConnectionRequestView to cover creation of connection request,107,129,DucPhamTV,12/16/20,1000,FALSE
```
Comment (Sulaiman001) [inline csvs/tasks.csv:466]: @Rajputusman 
Reviewed

**Comment by odetolataiwo** [comment]
sorry for the late responses. Holiday mood already. It is good to go now. Just resolve that conflict. I guess there was an update after this PR. Just add the details again and make the new PR.
@Sulaiman001 @Rajputusman

**Comment by Sulaiman001** [comment]
No problem @odetolataiwo Done check it again

**Pull request merged**